gensim/models/wrappers/ldamallet.py

```diff
@@ -268,11 +271,16 @@ def train(self, corpus):
         cmd = self.mallet_path + ' train-topics --input %s --num-topics %s  --alpha %s --optimize-interval %s '\
             '--num-threads %s --output-state %s --output-doc-topics %s --output-topic-keys %s '\
             '--num-iterations %s --inferencer-filename %s --doc-topics-threshold %s'
+
+        if self.random_seed != None:
+            cmd += ' --random-seed ' + str(self.random_seed)
+        
         cmd = cmd % (
             self.fcorpusmallet(), self.num_topics, self.alpha, self.optimize_interval,
             self.workers, self.fstate(), self.fdoctopics(), self.ftopickeys(), self.iterations,
             self.finferencer(), self.topic_threshold
         )
+
```

Comment:
why this?

Reply:
Actually I just inserted a blank line after the cmd definition just for readability - it is not required